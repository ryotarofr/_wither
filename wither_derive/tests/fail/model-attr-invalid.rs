use serde::{Serialize, Deserialize};
use wither::Model;

#[derive(Serialize, Deserialize, Model)]
#[model=""]
struct BadModel {
    #[serde(rename="_id", skip_serializing_if="Option::is_none")]
    pub id: Option<wither::bson::oid::ObjectId>,
}

fn main() {}
