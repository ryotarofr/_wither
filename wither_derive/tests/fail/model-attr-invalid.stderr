error: wither expected this attribute to be formatted as a meta list, eg: `#[model(...)]`
 --> $DIR/model-attr-invalid.rs:5:3
  |
5 | #[model=""]
  |   ^^^^^^^^
